tests/integration.rs

```diff
@@ -2291,3 +2291,9 @@ fn test_builtin_functions_context() {
         )))
     );
 }
+
+#[test]
+fn test_hex() {
```

Comment:
Should also test negative numbers in this test, as well as the string "0x" (which should return an error).

Reply:
Which error would be appropriate for the "0x" case? Currently it falls back to `EvalexprError::VariableIdentifierNotFound("0x")`.